ext/standard/tests/math/base_convert_variation1.phpt

```diff
@@ -71,7 +71,12 @@ $inputs = array(
 $iterator = 1;
 foreach($inputs as $input) {
 	echo "\n-- Iteration $iterator --\n";
-	var_dump(base_convert($input, 10, 8));
+    try {
+        $output = base_convert($input, 10, 8);
+    } catch (InvalidArgumentException $e) {
+        $output = 'INVALID';
```

Comment:
Please use the standard way of ``echo $e->getMessage() . \PHP_EOL`` (or can replace the PHP_EOL constant by ``"\n"``)